core/src/main/java/org/jdbi/v3/core/argument/BuiltInArgumentFactory.java

```diff
@@ -70,9 +70,9 @@
     @Override
     public Collection<? extends Type> prePreparedTypes() {
         return FACTORIES.stream()
-                .map(ArgumentFactory.Preparable::prePreparedTypes)
```

Comment:
This looks like a spacing-only change, should it be removed from the PR?

Reply:
collateral no-consequences formatting correction, may as well merge it? Or are 8-space indents not accidents?

Reply:
Well, that's what my IDE wants to do.  As soon as you change it to 4, if I edit the file and update the indentation, Eclipse changes it right back :/ so we end up in format edit wars.  That's part of why I don't like seeing format changes included in otherwise unrelated MRs, it tends to introduce unnecessary conflicts.

Reply:
The file's inconsistent to start with though, so maybe it's worth fixing it here.

Reply:
I agree on avoiding formatting wars, but yeah, this and most files use 4-space indents even in these structures, so this (and your eclipse's behavior) seems like a fluke to me. I'll take this up separately from this PR then.